obfuscate_and_normalize_test.go

```diff
@@ -92,7 +92,7 @@ func TestObfuscationAndNormalization(t *testing.T) {
 		},
 		{
 			input: `
-			/* this is a 
+			/* this is a ` + `
```

Comment:
This trailing space was driving me crazy as `go fmt` constantly removes it. Changing it to string concatenation so linters and formatters don't complain